### DISCUSSION TITLE
server header check

### DISCUSSION THREAD

**Discussion by strachotao** [matteocorti/check_ssl_cert#410] [Ideas]
I'd like to ask for a couple of implementations:
1) server header control... in the server header you don't want to have the version of the webserver, or even the name of the webserver e.g. "Apache". so something like --require-server-with-no-version or --require-server-string "myServer"
2) something like --require-no-x-powered-by to verify that this header is not present
3) something like --require-no-aspnet-version to verify that this header is not present

thank you for your work anyway :)

**Comment by matteocorti** [comment]
I created a new issue with the feature request and moved my answer there (so that it can be seen as part of the issue).
Thanks for the suggestion!

**Comment by matteocorti** [comment]
I implemented the feature. Please check the new README file

**Reply by strachotao** [reply]
Thank you ver much :)
